test/test_admin_integration.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from kafka import KafkaAdminClient
+from kafka.admin import ConfigResource, ConfigResourceType
+from test.fixtures import ZookeeperFixture, KafkaFixture
+from test.testutil import KafkaIntegrationTestCase, kafka_versions
+
+
+class TestAdminClientIntegration(KafkaIntegrationTestCase):
+    @classmethod
+    def setUpClass(cls):  # noqa
+        if not os.environ.get('KAFKA_VERSION'):
+            return
+
+        cls.zk = ZookeeperFixture.instance()
+        cls.server = KafkaFixture.instance(0, cls.zk)
+
+    @classmethod
+    def tearDownClass(cls):  # noqa
+        if not os.environ.get('KAFKA_VERSION'):
+            return
+
+        cls.server.close()
+        cls.zk.close()
+
+    def admin_client(self, **configs):
+        brokers = '%s:%d' % (self.server.host, self.server.port)
+        admin_client = KafkaAdminClient(
+            bootstrap_servers=brokers, **configs)
+        return admin_client
+
+    @kafka_versions('>=0.10.0')
```

Comment:
Use `pytest.mark.skipif()` instead of `@kafka_versions()`. Details in https://github.com/dpkp/kafka-python/pull/1887